src/Cubes/Cubes.jl

```diff
@@ -510,7 +510,11 @@ function DD.show_after(io::IO, mime, c::YAXArray)
     blockwidth = get(io, :blockwidth, 0)
     DD.print_block_separator(io, "file size", blockwidth, blockwidth)
     println(io, " ")
-    println(io, "  file size: ", formatbytes(cubesize(c)))
+    try
```

Comment:
I don't like the use of try catch in this.
Could we use Base.summarysize(first(c)) instead of sizeof(eltype(c))  in the definition of cubesize?
This should give the same result for plain types like Int and Float64 and gives an approximate size for other types like string.

Reply:
The problem is, that we would introduce a getindex call which might be even more problematic than the use of try catch. So I am fine with that or with removing this info and only enabling that for certain eltypes where we know that sizeof is giving the right answer.
We could do that by putting the lines 511 to 513 together into a print_cubesize function which can then dispatch to the eltype of the cube and which can default to nothing to not print this information.

Reply:
> The problem is, that we would introduce a getindex call which might be even more problematic than the use of try catch. So I am fine with that or with removing this info and only enabling that for certain eltypes where we know that sizeof is giving the right answer. We could do that by putting the lines 511 to 513 together into a print_cubesize function which can then dispatch to the eltype of the cube and which can default to nothing to not print this information.

Thanks for the response and sorry for the delay.

I made a change checking for some specific eltypes for which the function `sizeof` is known to behave well before printing the size of the datacube. Let me know if that approach is better or what you think I need to change.